components/ModEditSubmissionForm.vue

```diff
@@ -855,6 +870,17 @@ async function submitCompletedForm(e: Event) {
 
 const syntheticEvent = new Event('submit', { bubbles: true, cancelable: true })
 
+const resetModalRefs = async () => {
+    moderationSubmissionStore.setShowRejectSubmissionConfirmation(false)
+    moderationSubmissionStore.setApprovingSubmissionFromTopBar(false)
+}
+
+const rejectSubmission = async () => {
```

Comment:
should we show a loading indicator here while it's being rejected? We have our loader component we could use

Reply:
That's a good idea! What do you think about showing something like a  "Sending your request" loader? I think it would be better for the user since we aren't actually waiting to load the next page, but we're waiting for the request to be completed.

Reply:
sounds like a good next issue! can you make one for it?

Reply:
@ermish Here's the ticket: https://github.com/ourjapanlife/findadoc-web/issues/884